nginx/test/ci/fixtures/plus_api_slabs.json

```diff
@@ -0,0 +1,808 @@
+{
+    "SSL": {
```

Comment:
Does it make sense to pass this as a tag ?

Reply:
I believe so yes. As you can read [here](http://nginx.org/en/docs/http/ngx_http_status_module.html) in the `slabs` description, there are the same metrics "for each shared memory zone that uses slab allocator", so this shared memory zone should appear as a tag.

Reply:
#1053